services/collectd/service.go

```diff
@@ -17,11 +19,16 @@ import (
 	"collectd.org/network"
 	"github.com/influxdata/influxdb/logger"
 	"github.com/influxdata/influxdb/models"
+	"github.com/influxdata/influxdb/services"
 	"github.com/influxdata/influxdb/services/meta"
 	"github.com/influxdata/influxdb/tsdb"
 	"go.uber.org/zap"
 )
 
+func init() {
+	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
We don't normally set none-default logging flags as far as I know. Any particular reason for this?

Reply:
this is stray code.  i did this for testing and forgot about it.  will remove.